src/test/unit/comparison.test.js

```diff
@@ -30,4 +30,42 @@ describe('mergeProfiles function', function() {
 
     expect(mergedProfile.profile.meta.interval).toEqual(10);
   });
+
+  it('should set the resulting profile to symbolicated if all are symbolicated', () => {
+    const { profile: symbolicatedProfile } = getProfileFromTextSamples(`A`);
+    const { profile: unsymbolicatedProfile } = getProfileFromTextSamples(`B`);
+    unsymbolicatedProfile.meta.symbolicated = false;
```

Comment:
I think these tests are missing the unknown symbolication case here.

Reply:
right !

Reply:
And thanks to the added tests I found problems in the added code for this case...